english/coffee/coffee.md

```diff
@@ -44,3 +44,10 @@ After having your coffee do you want to:
 
 Coffee creates a great medium for people to have conversations about ideas and visions.
 Have chocolate coffee in the morning and [enjoy.](another-coffee/another-coffee.md) 
+
+I always love eating donuts with my coffee. but I don't have any donuts so I drive to the local donut shop to buy some.
+On the way, I get stopped by a police officer.  I pull to the side and the cop knocks on my window and asks for my license
+and registration.  
+
+[pull out wallet](wallet/wallet.md)
+[invite him to the donut shop](inviteCop/inviteCop.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.